coralproject/talk
# Deep Threading Support

## What does this PR do?
- Support deep nested comment
- Support configuring nesting level using ENV variable  `TALK_THREADING_LEVEL`
- Currently defaults to `3` thread levels (I chose that for easy testing, we'll need to define this)
- Includes styling for up to `3` thread levels

#### New global CSS classNames
`talk-stream-comment-wrapper-level-${depth}`
`talk-stream-comment`
`talk-stream-comment-level-${depth}`
`talk-stream-highlighted-comment`
`talk-stream-pending-comment`

#### Changed CSS classNames
`coral-new-comments` -> `talk-new-comments`

## How do I test this PR?

- Check all processes that includes comments like (posting, replying, live update, etc..)
- Change `TALK_THREADING_LEVEL` to a different count.
